prisma/schema.prisma

```diff
@@ -43,10 +43,11 @@ model Comment {
 }
 
 model Post {
-  id    String @id @default(cuid())
+  id    String  @id @default(cuid())
```

Comment:
remove space

Reply:
Because Prisma formatter is installed, it does this formatting